src/pages/ContributorFields.tsx

```diff
@@ -92,72 +88,72 @@ const ContributorFields: FC<ContributorFieldsProps> = ({ setAllChangesSaved }) =
   };
 
   return (
-    <StyledPaper>
-      <Typography variant="h1">{t('resource.metadata.contributors')}</Typography>
```

Comment:
Trenger vi ikke tittelen? Det skal også være en "opphaver" felt også på den samme stepper-fanen.

Reply:
ah. det var tittelen for feltsamlingen. Da skal den ikke slettes nei. Kanskje bare kjøres ned litt i størrelse

Reply:
fixed